robowflex_ompl/include/robowflex_ompl/ompl_trajectory.h

```diff
@@ -53,6 +53,13 @@ namespace robowflex
              *  \return The Geometric Path equivalent of the trajectory.
              */
             ompl::geometric::PathGeometric toOMPLPath(const ompl::geometric::SimpleSetupPtr &ss);
+
+            /** \brief Sets the trajectory from an OMPL Path and a reference state.
+             *  \param[in] reference_state A full state that contains the values for all the joints.
```

Comment:
Would it not be reasonable to have the OMPL trajectory class itself contain the reference state for the trajectory? This trajectory was generated with respect to some default state. This reference state could be the zero state by default. Is there a case where we want an OMPL trajectory and do not have a reference state (even implicitly)?

Reply:
This is a big change making it an issue,